aten/src/ATen/native/transformers/attention.cpp

```diff
@@ -495,6 +496,58 @@ inline void validate_sdpa_input(
   }
   return;
 }
+
+double get_min_value(caffe2::TypeMeta dtype){
```

Comment:
I wrestled tooooooooooooooo long trying to find the pattern for this. There does exists a ScaleTypetoCPPType I thought I had something but was having trouble